js/src/wcdl/select-control/index.scss

```diff
@@ -1,13 +1,12 @@
 .wcdl-select-control {
-	&__label,
 	&__input,
 	&__helper-text {
 		margin-bottom: calc(var(--main-gap) / 3);
 	}
 
 	&__label {
-		font-size: 13px;
-		line-height: 16px;
+		color: $gray-700;
+		padding-bottom: 4px;
```

Comment:
❔ Looks like it would be more proper to use `margin-bottom` since it's for spacing to the input under itself?

Reply:
I used `padding-bottom` because I was referring to InputControl and they use `padding-bottom` too, so I think it might be better to be consistent with other input controls. 

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/417342/156432628-b1de8093-b351-458d-a429-9c95623d73e9.png">

Reply:
Got it, thanks!